pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -223,6 +223,7 @@ def cmd(self):
             "BOK_CHOY_HAR_DIR='{}'".format(self.har_dir),
             "BOKCHOY_A11Y_CUSTOM_RULES_FILE='{}'".format(self.a11y_file),
             "SELENIUM_DRIVER_LOG_DIR='{}'".format(self.log_dir),
+            "VERIFY_XSS='True'",
```

Comment:
Note that this is just a hack to turn on the bok choy testing. I will be adding a proper argument that can be specified when executing the suite.